packages/desktop-client/src/components/reports/graphs/CalendarGraph.tsx

```diff
@@ -0,0 +1,314 @@
+                              </PrivacyFilter>
+                            ) : (
+                              ''
+                            )}
+                          </View>
+                          <View style={{ marginLeft: 4, flexDirection: 'row' }}>
+                            (
+                            <PrivacyFilter>
+                              {Math.round(day.expenseSize * 100) / 100 + '%'}
+                            </PrivacyFilter>
+                            )
+                          </View>
+                        </>
+                      )}
+                    </View>
+                  </View>
+                </View>
+              }
+              placement="bottom end"
+              style={{
+                ...styles.tooltip,
+                lineHeight: 1.5,
+                padding: '6px 10px',
+              }}
+            >
+              <DayButton
+                key={day.date.getTime()}
+                resizeRef={index === 15 ? buttonRef : null}
+                fontSize={fontSize}
+                day={day}
+                onPress={() => onDayClick(day.date)}
+              />
+            </Tooltip>
+          ) : (
+            <DayButton
+              key={day.date.getTime()}
+              resizeRef={index === 15 ? buttonRef : null}
+              fontSize={fontSize}
+              day={day}
+              onPress={() => onDayClick(day.date)}
+            />
+          ),
+        )}
+      </View>
+    </>
+  );
+}
+
+type DayButtonProps = {
+  fontSize: number;
+  resizeRef: Ref<HTMLButtonElement>;
+  day: {
+    date: Date;
+    incomeSize: number;
+    expenseSize: number;
+  };
+  onPress: () => void;
+};
+function DayButton({ day, onPress, fontSize, resizeRef }: DayButtonProps) {
+  const [currentFontSize, setCurrentFontSize] = useState(fontSize);
+
+  useEffect(() => {
+    setCurrentFontSize(fontSize);
+  }, [fontSize]);
+
+  return (
+    <Button
+      ref={resizeRef}
+      aria-label={format(day.date, 'MMMM d, yyyy')}
+      style={{
+        borderColor: 'transparent',
+        backgroundColor: theme.calendarCellBackground,
+        position: 'relative',
+        padding: 'unset',
+        height: '100%',
+        minWidth: 0,
+        minHeight: 0,
+        margin: 0,
+      }}
+      onPress={() => onPress()}
+    >
+      {day.expenseSize !== 0 && (
+        <View
+          style={{
+            position: 'absolute',
+            width: '50%',
+            height: '100%',
+            background: chartTheme.colors.red,
+            opacity: 0.2,
+            right: 0,
+          }}
+        />
+      )}
+      {day.incomeSize !== 0 && (
+        <View
+          style={{
+            position: 'absolute',
+            width: '50%',
+            height: '100%',
+            background: chartTheme.colors.blue,
+            opacity: 0.2,
+            left: 0,
+          }}
+        />
+      )}
+      <View
+        className="bar positive-bar"
```

Comment:
We should just use `className` + emotion's `css` here to merge classnames and styles in one declaration

Reply:
this is a left out from testing classes, removed the classnames